waku/waku_test.go

```diff
@@ -735,12 +742,18 @@ func TestSymmetricSendCycleWithTopicInterest(t *testing.T) {
 	InitSingleTest()
 
 	w := New(nil, nil)
-	defer w.SetMinimumPoW(common.DefaultMinimumPoW, false)  // nolint: errcheck
-	defer w.SetMaxMessageSize(common.DefaultMaxMessageSize) // nolint: errcheck
+	defer func() {
+		handleError(t, w.SetMinimumPoW(common.DefaultMinimumPoW, false))
+	}()
+	defer func() {
+		handleError(t, w.SetMaxMessageSize(common.DefaultMaxMessageSize))
+	}()
 	if err := w.Start(nil); err != nil {
 		t.Fatal("could not start node")
 	}
-	defer w.Stop() // nolint: errcheck
+	defer func() {
+		handleError(t, w.Stop())
+	}()
```

Comment:
[Error handling is good](https://github.com/status-im/status-go/pull/1957/files#r417328424)